### ISSUE TITLE
Ultralytics rtdetr-l.pt 转换ncnn失败

### ISSUE THREAD

**Issue by lxyjyy** [pnnx/pnnx#134]
pnnx 版本:20240102

rtdetr-l.pt 下载路径: https://github.com/ultralytics/assets/releases/download/v8.2.0/rtdetr-l.pt

报错日志如下:
pnnxparam = rtdetr_l.pnnx.param
pnnxbin = rtdetr_l.pnnx.bin
pnnxpy = rtdetr_l_pnnx.py
pnnxonnx = rtdetr_l.pnnx.onnx
ncnnparam = rtdetr_l.ncnn.param
ncnnbin = rtdetr_l.ncnn.bin
ncnnpy = rtdetr_l_ncnn.py
fp16 = 1
optlevel = 2
device = cpu
inputshape = [1,3,640,640]f32
inputshape2 = 
customop = 
moduleop = 
############# pass_level0
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.modules.block.HGBlock
inline module = ultralytics.nn.modules.block.HGStem
inline module = ultralytics.nn.modules.block.RepC3
inline module = ultralytics.nn.modules.conv.Concat
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.conv.LightConv
inline module = ultralytics.nn.modules.conv.RepConv
inline module = ultralytics.nn.modules.head.RTDETRDecoder
inline module = ultralytics.nn.modules.transformer.AIFI
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoder
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoderLayer
inline module = ultralytics.nn.modules.transformer.MLP
inline module = ultralytics.nn.modules.transformer.MSDeformAttn
inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.modules.block.HGBlock
inline module = ultralytics.nn.modules.block.HGStem
inline module = ultralytics.nn.modules.block.RepC3
inline module = ultralytics.nn.modules.conv.Concat
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.conv.LightConv
inline module = ultralytics.nn.modules.conv.RepConv
inline module = ultralytics.nn.modules.head.RTDETRDecoder
inline module = ultralytics.nn.modules.transformer.AIFI
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoder
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoderLayer
inline module = ultralytics.nn.modules.transformer.MLP
inline module = ultralytics.nn.modules.transformer.MSDeformAttn

----------------

inline module = torch.nn.modules.linear.Identity
inline module = ultralytics.nn.modules.block.HGBlock
inline module = ultralytics.nn.modules.block.HGStem
inline module = ultralytics.nn.modules.block.RepC3
inline module = ultralytics.nn.modules.conv.Concat
inline module = ultralytics.nn.modules.conv.Conv
inline module = ultralytics.nn.modules.conv.DWConv
inline module = ultralytics.nn.modules.conv.LightConv
inline module = ultralytics.nn.modules.conv.RepConv
inline module = ultralytics.nn.modules.head.RTDETRDecoder
inline module = ultralytics.nn.modules.transformer.AIFI
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoder
inline module = ultralytics.nn.modules.transformer.DeformableTransformerDecoderLayer
inline module = ultralytics.nn.modules.transformer.MLP
inline module = ultralytics.nn.modules.transformer.MSDeformAttn

----------------

############# pass_ncnn
force batch axis 233 for operand 242
force batch axis 233 for operand 239
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
binaryop tensor0 with rank 6 is not supported yet!
binaryop tensor1 with rank 6 is not supported yet!
stack along batch axis 0 is not supported
stack along batch axis 0 is not supported
squeeze batch dim 0 is not supported yet!
squeeze batch dim 0 is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
permute across batch dim is not supported yet!
ignore torch.topk torch.topk_386 param dim=1
ignore torch.topk torch.topk_386 param k=300
ignore torch.topk torch.topk_386 param largest=True
ignore torch.topk torch.topk_386 param sorted=True
ignore pnnx.Expression pnnx_expr_1452 param expr=[@0,@1]
ignore pnnx.Expression pnnx_expr_1444 param expr=[None,@0]
ignore torch.flatten torch.flatten_311 param end_dim=1
ignore torch.flatten torch.flatten_311 param start_dim=0
ignore torch.flatten torch.flatten_309 param end_dim=1
ignore torch.flatten torch.flatten_309 param start_dim=0
ignore torch.flatten torch.flatten_307 param end_dim=1
ignore torch.flatten torch.flatten_307 param start_dim=0
ignore torch.flatten torch.flatten_312 param end_dim=-1
ignore torch.flatten torch.flatten_312 param start_dim=-2
ignore torch.flatten torch.flatten_318 param end_dim=1
ignore torch.flatten torch.flatten_318 param start_dim=0
ignore torch.flatten torch.flatten_316 param end_dim=1
ignore torch.flatten torch.flatten_316 param start_dim=0
ignore torch.flatten torch.flatten_314 param end_dim=1
ignore torch.flatten torch.flatten_314 param start_dim=0
ignore torch.flatten torch.flatten_319 param end_dim=-1
ignore torch.flatten torch.flatten_319 param start_dim=-2
ignore torch.flatten torch.flatten_325 param end_dim=1
ignore torch.flatten torch.flatten_325 param start_dim=0
ignore torch.flatten torch.flatten_323 param end_dim=1
ignore torch.flatten torch.flatten_323 param start_dim=0
ignore torch.flatten torch.flatten_321 param end_dim=1
ignore torch.flatten torch.flatten_321 param start_dim=0
ignore torch.flatten torch.flatten_326 param end_dim=-1
ignore torch.flatten torch.flatten_326 param start_dim=-2
ignore torch.flatten torch.flatten_332 param end_dim=1
ignore torch.flatten torch.flatten_332 param start_dim=0
ignore torch.flatten torch.flatten_330 param end_dim=1
ignore torch.flatten torch.flatten_330 param start_dim=0
ignore torch.flatten torch.flatten_328 param end_dim=1
ignore torch.flatten torch.flatten_328 param start_dim=0
ignore torch.flatten torch.flatten_333 param end_dim=-1
ignore torch.flatten torch.flatten_333 param start_dim=-2
ignore torch.flatten torch.flatten_339 param end_dim=1
ignore torch.flatten torch.flatten_339 param start_dim=0
ignore torch.flatten torch.flatten_337 param end_dim=1
ignore torch.flatten torch.flatten_337 param start_dim=0
ignore torch.flatten torch.flatten_335 param end_dim=1
ignore torch.flatten torch.flatten_335 param start_dim=0
ignore torch.flatten torch.flatten_340 param end_dim=-1
ignore torch.flatten torch.flatten_340 param start_dim=-2
ignore torch.flatten torch.flatten_346 param end_dim=1
ignore torch.flatten torch.flatten_346 param start_dim=0
ignore torch.flatten torch.flatten_344 param end_dim=1
ignore torch.flatten torch.flatten_344 param start_dim=0
ignore torch.flatten torch.flatten_342 param end_dim=1
ignore torch.flatten torch.flatten_342 param start_dim=0
ignore torch.flatten torch.flatten_347 param end_dim=-1
ignore torch.flatten torch.flatten_347 param start_dim=-2
ignore Concat stack_6 param dim=0
ignore Concat stack_7 param dim=0
